setup.py

```diff
@@ -3,31 +3,30 @@
 from setuptools import setup, find_packages
 
 
+TESTS_REQUIRE = ['zope.testing', 'zope.testrunner']
+
 setup(
     name='gocept.filestore',
     version='0.4dev',
     author='gocept gmbh & co. kg',
     author_email='mail@gocept.com',
-    url='https://bitbucket.org/gocept/gocept.filestore',
+    url='https://github.com/gocept/gocept.filestore',
     description="Provides maildir like access to files",
```

Comment:
It would be good to have some trove classifier here to indicate at least the supported Python versions and some areas of application.